docs/features/create-custom-events.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Create custom events"
+layout: default
+---
+
+## Create Custom Events
+Besides the [offcial Azure SDK event schemas](https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.eventgrid.models?view=azure-dotnet), we support custom events also.
```

Comment:
Let's highlight our work first over the built-in schemas to make it more discoverable
```suggestion
We allow you to create custom events next to the [official Azure SDK event schemas](https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.eventgrid.models?view=azure-dotnet).
```